README.md

```diff
@@ -1 +1,18 @@
-# talks
\ No newline at end of file
+# Flutter Jakarta Talks
+
+We want Flutter Jakarta to be a place for flutter enthusiasts to share knowledge about development with Flutter. This repository is intended as an archive and talks proposal for anyone who wants to be a speaker at our monthly meetup. We usually hold meetups in the last week every 1 month (if possible).
+
+## Submit Your Talks Proposal
+
+Please submit your talk proposal through github issues, you have 30 minutes to bring the talk you want. Make sure your talk proposal includes the following information:
```

Comment:
`Please submit your talk proposal by creating a GitHub Issue. You have 30 minutes to present, and make sure that your talk proposal includes the following information: `